### PR TITLE
[RU] osu!api

### PR DESCRIPTION
**Pull request by vp2woon**

### CODE REVIEW

File: wiki/osu!api/ru.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+У osu! есть публичный API, который сторонние сервисы могут использовать для расширения и поддержки osu! платформы. Раньше он был более приватным, использовался только для каждого запроса, но теперь любой может использовать API, если у него есть учетная запись osu!.
+
+Вы можете найти документацию по API в [osu!api Вики] (https://github.com/ppy/osu-api/wiki). Текущая версия API использует закрытые ключи для аутентификации запросов.
```
Comment (cl8n) [inline wiki/osu!api/ru.md]: the spaces between `]` and `(` need to be removed for the link to render (same goes for every other link here)

File: wiki/osu!api/ru.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+У osu! есть публичный API, который сторонние сервисы могут использовать для расширения и поддержки osu! платформы. Раньше он был более приватным, использовался только для каждого запроса, но теперь любой может использовать API, если у него есть учетная запись osu!.
```
Comment (cl8n) [inline wiki/osu!api/ru.md]: missing title before this, in english it has
```markdown
# osu!api
```

Comment (vp2woon) [reply wiki/osu!api/ru.md]: oh, sure

File: wiki/osu!api/ru.md (hunk 1 above)
Comment (vp2woon) [reply wiki/osu!api/ru.md]: yes, thanks

**Pull request merged**